cypress/integration/chi-custom-elements/carousel.spec.js

```diff
@@ -216,4 +217,88 @@ describe('Carousel', () => {
       cy.get('@prevButton').should('be.disabled');
     });
   });
+
+  describe('No Button Controllers', () => {
+    beforeEach(() => {
+      cy.get("[data-cy='no-button-controllers']").as('no-button-controllers');
+    });
+
+    it('Should have attribute no-button-controllers', () => {
+      cy.get('@no-button-controllers').should(
+        'have.attr',
+        'no-button-controllers'
+      );
+    });
+
+    it('should not have direction buttons controller', () => {
+      cy.get('@no-button-controllers')
+        .find(CAROUSEL_DIRECTION_CONTROL_CLASS)
+        .should('not.exist');
+    });
+  });
+
+  describe('Autoplay', () => {
+    const interval = 3000;
+
+    beforeEach(() => {
+      cy.get("[data-cy='autoplay']").as('autoplay');
+      cy.get(`.${CAROUSEL_ITEM_CLASS}.${ACTIVE_CLASS}`).as('activeItems');
+    });
+
+    it('Should have attribute autoplay', () => {
+      cy.get('@autoplay').should('have.attr', 'autoplay');
+    });
+
+    it('Should have attribute interval', () => {
+      cy.get('@autoplay').should('have.attr', 'interval', 3);
+    });
+
+    it('Should autoplay with 3 seconds interval', () => {
+      cy.visit('tests/custom-elements/carousel.html');
```

Comment:
Do we need this `cy.visit`?

Reply:
@josecarlosgonzalezv unfortunately this is needed to reset the carousel to the first position. 
Another options: 
- Move this to be the first test
- Implement a way to set the carousel position (with a huge cost)